src/modules/chat_deleted_messages/index.js

```diff
@@ -38,11 +38,10 @@ function onClearChat(name, tags) {
                 });
 
                 if (!settings.get('showDeletedMessages')) {
-                    $message.addClass('bttv-click');
                     const $messageClone = $message.clone();
+                    $message.addClass('bttv-click');
                     $message.click(() => {
                         $message.replaceWith($messageClone);
-                        $message.removeClass('bttv-click');
```

Comment:
I switch the order of the clone creation so the `bttv-click` class is added **after** the clone is created. No need to remove the class, since it wasn't there when the clone was created.